ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -59,5 +102,321 @@ private static CCDPaymentIntention getCCDPaymentIntention() {
             .build();
     }
 
+    public static CCDAddress getCCDAddress() {
+        return CCDAddress.builder()
+            .addressLine1("line1")
+            .addressLine2("line2")
+            .addressLine3("line3")
+            .postTown("city")
+            .postCode("postcode")
+            .build();
+    }
+
+    public static CCDDefendant getCCDDefendantIndividual() {
+        CCDAddress ccdAddress = getCCDAddress();
+        return CCDDefendant.builder()
+            .claimantProvidedType(INDIVIDUAL)
+            .claimantProvidedAddress(ccdAddress)
+            .claimantProvidedName("Individual")
+            .claimantProvidedDateOfBirth(LocalDate.of(1950, 01, 01))
+            .claimantProvidedServiceAddress(ccdAddress)
+            .representativeOrganisationAddress(ccdAddress)
+            .representativeOrganisationName("My Org")
+            .representativeOrganisationPhone("07987654321")
+            .representativeOrganisationEmail(",my@email.com")
```

Comment:
"Please remove comma from, I know this is plain text, but does not look clean."  -- your comment from another PR. what is the point in giving this comment when you have blatantly committed this code. Instead of wasting time in repeating this comment in the other PR you could have used an intellij shortcut to find and replace. The short cut is ctrl+R